pkg/sql/logictest/testdata/logic_test/srfs

```diff
@@ -920,15 +935,15 @@ vals       2
 vals       3
 
 query TT colnames
-SELECT relname, information_schema._pg_expandarray(indkey) FROM pg_class, pg_index WHERE pg_class.oid = pg_index.indrelid ORDER BY relname, x, n
+SELECT relname, information_schema._pg_expandarray(indkey) FROM pg_class, pg_index WHERE pg_class.oid = pg_index.indrelid ORDER BY relname, indexrelid, x, n
```

Comment:
just a sanity check: what is `x` and `n` referring to here? (i tried running this example in postgres, and got `column "x" does not exist`